Proven-In-Use-Argument.md

```diff
@@ -0,0 +1,77 @@
+# Is Linux mature and proven enough for use in safety-critical applications?
+
+Linux has proven itself thousands, probably millions of times over in business-critical applications. So it makes sense to use Linux for safety-critical applications as well. Linux has many advantages 
```

Comment:
There are several misconceptions already at the beginning:

1. "Linux", as intended in the document, doesn't exist. There are a large number of versions and revisions, each with own set of features and bugs.
2. When making claims of proven in use, one MUST refer to a specific SW configuration, its features and its bugs.
3. When making claims about proven in use, one MUST refer to a specific HW configuration.

There would be even the aspect of the toolchain, but the previous points are already sufficient to void the rest of the claims.

As I already stated, these sort of statements are not per se comepltely wrong, but they do nto belong to "Proven in Use", they rather fit in the PAS 8926 https://www.iso.org/standard/83346.html approach.

Reply:
I still do not accept to give up the option of PiU alltogether. Can you agree with the following text? Would this make the problem sufficiently clear and still provide a constructive path for the argument?

"""
It is obvious and must nevertheless be clearly repeated: Linux can never be approved 'as a whole' and 'in general' for use in safety-critical applications. There are any number of examples of Linux configurations and constellations in which the operating system fails mercilessly. Therefore, according to the conservative interpretation of the standard documents, only individual, fully specified systems can prove themselves. In this sense, every Linux system is unique and a PiU argument is categorically ruled out. 

In order to approach the PiU argumentation for Linux, the dimension of the system under consideration or the elements under consideration must be reduced and focused on the one hand. On the other hand, value ranges for configuration and constellation must be described that are recognized as equivalent with regard to the safety-critical application. If it is possible to document the elements in focus in running systems with their behavior, especially in the event of system errors and in extreme situations, and to document robust behavior, a PiU argumentation could be based on this.
"""

Reply:
In my opinion, the initial problem lies in the wording "Proven in Use" - it is already widely asociated to certain concepts which do not match what you are after. I do not understand why you seem unmovable on this.
Of course you can arbitrarily redefine words and expressions, to mean what you wish them to mean.
However, it seems an unnecessary uphill battle.

Then, there are a bunch of further considerations on several assumptions that you seem to make, like what is the "installed base" that can be taken as reference, what is the accept/reject criteria.

For example: a handheld console, a water plant management system, a server,  a drone, can all run linux.
How do you define which ones can be taken as reference and which cannot?
Assuming that you have defined such a set, which changes can one make and still keep the claims valid and which would void the claims.

To me, this whole argumentation looks very verbose, but it lacks the clarity of scope that I would expect when making such claims.

It is a fact that - usually - one will want to take something as baseline and alter it.

This whole thing is missing clear criteria for what is admissible, what is not, and what is admissible, but introduces extra validation (for example).

I'd rather see a set of conicise bullet points with ok/nok do/don't and so on.